packages/zoe/src/zoeService/instanceAdminStorage.js

```diff
@@ -182,7 +182,7 @@ const makeInstanceAdminBehavior = (zoeBaggage, makeZoeSeatAdminKit) => {
         /** @param {HandleOfferResult} result */
         result => zoeSeatAdmin.resolveExitAndResult(result),
         err => {
-          state.adminNode.terminateWithFailure(err);
+          E(state.adminNode).terminateWithFailure(err);
```

Comment:
This looks like an ungrounded promise, does that matter? (if it helps, `terminateWithFailure` will execute promptly, and the only reason it might fail is if `err` could not be serialized or held a Promise)